src/locationsettings.cpp

```diff
@@ -515,7 +515,7 @@ void LocationSettings::setGpsFlightMode(bool flightMode)
     if (d->m_gpsTechInterface) {
         QDBusReply<void> reply = d->m_gpsTechInterface->blockingCall("SetProperty",
                                                              PoweredPropertyName,
-                                                             QVariant::fromValue<QDBusVariant>(QDBusVariant(QVariant::fromValue<bool>(!flightMode))));
+                                                             QVariant::fromValue<bool>(!flightMode));
```

Comment:
Didn't test, but wouldn't this now call SetProperty with bool as parameter type instead of dbus variant bool?

Reply:
Good point, I need to test. Maybe that's the reason of such complex construction to pass a boolean value in the end.

Reply:
Long time. Could just close the pr maybe?

Reply:
Yes, indeed.